Assets/Scripts/Boids/BoidObstacleProxy.cs

```diff
@@ -0,0 +1,12 @@
+using Unity.Entities;
+using Unity.Transforms;
+using UnityEngine;
+
+namespace Boids
+{
+    public struct BoidObstacle : IComponentData
+    { }
```

Comment:
This struct is essentially a tag to label an boid (Entity) as an Obstacle. This plays into steering as regular boid's will seek to avoid the obstacles.